app/facades/foods_facade.rb

```diff
@@ -0,0 +1,19 @@
+class FoodsFacade
+  def self.top_10_foods_searched
+    response = Faraday.get('https://api.nal.usda.gov/fdc/v1/foods/search') do |request|
+      request.params['api_key'] = 'nTNqChgK4a5uwh6kyqJ0B2NCCC9IovJRJT5oLrge'
+      request.params['query'] = "sweet potatoes"
+      request.params['ingredients'] = "sweet potatoes"
+    end
+
+    parsed = JSON.parse(response.body, symbolize_names: true)
+
+    foods = []
+
+    10.times do |i|
+      foods << Food.new(parsed[:foods][i])
+    end
+
+    foods
+  end
```

Comment:
I know this is a work in progress, and I'm glad to see you starting to pull this into a facade. We can think of your facade as the layer that organizes the data after it's been collected. So the code in here that collects data is everything that calls to the api and parses the information, and that's what should get moved to your service.